src/Spice86.Core/Emulator/Gdb/GdbIo.cs

```diff
@@ -33,10 +33,10 @@ public sealed class GdbIo : IDisposable {
     public void WaitForConnection() {
         _tcpListener.Start();
         _socket = _tcpListener.AcceptSocket();
-        if (_loggerService.IsEnabled(Serilog.Events.LogEventLevel.Information)) {
+        if (_loggerService.IsEnabled(Serilog.Events.LogEventLevel.Verbose)) {
             int port = ((IPEndPoint)_tcpListener.LocalEndpoint).Port;
-            _loggerService.Information("GDB Server listening on port {@Port}", port);
-            _loggerService.Information("Client connected: {@CanonicalHostName}", _socket.RemoteEndPoint);
```

Comment:
This seems like useful information to have, even when you're not specifically debugging the Gdb Server.

Reply:
Implemented.